src/file-utils.ts

```diff
@@ -25,6 +25,15 @@ export function createFolderSync(folder: string): void {
     }
 }
 
+export function getFileExtensionFromFileName(fullPath: string): string | undefined {

```

Comment:
You're already using the `path` module in this project. It also provides a `path.extname(path)` method. Is there a reason you implemented it yourself?

Reply:
yes right, let's use the path method.